packages/orbit-components/babel.config.js

```diff
@@ -0,0 +1,7 @@
+// @noflow
+const baseConfig = require("../../babel.config");
+
+module.exports = {
+  ...baseConfig,
+  presets: process.env.BABEL_TARGET === "storybook" ? [] : baseConfig.presets,
```

Comment:
Because Storybook is only used for `orbit-components` (for now), I moved that part of Babel config logic here.